src/api/auth.ts

```diff
@@ -407,7 +407,8 @@ export function publishPackage(
                 owners: [ username ],
                 organization: organization ? organization : '',
                 teamPermissions: permissions,
-                memberPermissions: [{ name: username, permission: 'read-write' }]
+                memberPermissions: [{ name: username, permission: 'read-write' }],
```

Comment:
your objection convinced me :) there's no reason to have `read: true` if its always the same value.  maybe just `permissions: true / false` in that case?